src/python/turicreate/toolkits/_pre_trained_models.py

```diff
@@ -224,10 +224,41 @@ def __init__(self):
     def get_model_path(self):
         return self.model_path
 
+class StyleTransferTransformerCoreML():
+
+    def __init__(self):
+        self.name = 'transformer'
```

Comment:
Underscore `_coreml`? Perhaps could be a problem.

Reply:
Yes.